src/LangtailPrompts.ts

```diff
@@ -143,4 +151,36 @@ export class LangtailPrompts {
     result.httpResponse = res
     return result
   }
+
+  async get({
+    prompt,
+    environment,
+    version,
+  }: {
+    prompt: string
+    environment: Environment
```

Comment:
environment should be optional and default to production like invoke

Reply:
fixed